index.js

```diff
@@ -196,9 +253,9 @@ function getGermanCars(/* code here */) {
  *   return num * 2
  * }
 */
-const sum = null; // code here!
-const addFive = null; // code here!
-const argTimesTwo = null; // code here!
+const sum = (a, b) => {return a + b;} 
+const addFive = (num) => {return num+5;}
+const argTimesTwo = (num) => { return num*2; }
 
```

Comment:
with arrow functions the return is implied.  So you can take away the curly braces and the return keyword as well.  You will only need the curly braces if it has more than one statement inside the function and in that case you will also need to use the return.